guides/welfare/raising_an_issue.md

```diff
@@ -28,9 +22,9 @@ You can also call the [Acas](http://www.acas.org.uk/index.aspx?articleid=2042) o
 
 At the time of writing we do not have anyone with mediation training at the company but if necessary we can bring in third party mediators to deal with disputes informally.
 
-You an find out more about Mediation from [Acas](http://www.acas.org.uk/index.aspx?articleid=1680).
+You can find out more about Mediation from [Acas](http://www.acas.org.uk/index.aspx?articleid=1680).
 
-If you wish to enter a mediation process, please ask a director or ask a colleague to go to a director to request mediation.
+If you wish to enter a mediation process, please ask the people team or ask a colleague to speak to the people team on your behalf to request mediation.
```

Comment:
Why would I ask a colleague to ask on my behalf? Can we make those situations more explicit?

Reply:
This is true. doesn't make sense. Maybe its your mentor who would ask on your behalf? Rather than a colleague

Or should we just remove that bit and just say 'please ask the people team to request mediation' ?

Reply:
I think it would be fine to suggest just asking the people team.

Reply:
Thank you Kyle. that's changed now